app/Models/Permission.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Permission extends Model
+{
+    use SoftDeletes;
+
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table = "permissions";
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Models/Permission.php</br>String "permissions" does not require double quotes; use single quotes instead